cypress/support/commands/rancher-api-commands.ts

```diff
@@ -622,3 +651,12 @@ Cypress.Commands.add('createAmazonMachineConfig', (instanceType, region, vpcId,
       expect(resp.status).to.eq(201);
     });
 });
+
+// update resource list view preference
+Cypress.Commands.add('updateResourceListViewPref', (clusterName: string, groupBy:string, namespaceFilter: string) => {
```

Comment:
one for another pr as well, this is the namespace filter which affects lists, side nav, some drop downs, etc. so something like  `updateNamespaceFilter` would be more suitable

Reply:
will address in the next pr